uninstall.php

```diff
@@ -11,4 +11,32 @@
 	exit;
 }
 
-delete_option( 'perflab_modules_settings' );
+if ( is_multisite() ) {
+	$site_ids = get_sites(
+		array(
+			'fields'                 => 'ids',
+			'number'                 => '',
```

Comment:
As Felix comment https://github.com/WordPress/performance/pull/345#issuecomment-1155608644

> What we could do is set a sensible limit, e.g. set number to 50 or something. Then it would least support the majority of multisites that don't have a ton of sites.

Reply:
I feel like this should support a higher number of sites, maybe 100? Also, should we consider checking the number of sites in the network before the query and if there are over X just skip trying to delete entirely?

Reply:
I agree with Adam, we should check and skip

Reply:
See my feedback in https://github.com/WordPress/performance/pull/458#discussion_r940206315, IMO it's not worth checking for the number of sites before the main site query. What is the harm of deleting the option for 100 sites when there are more than 100 sites? IMO it's a better outcome to the option removed on e.g. 100 out of 130 sites rather than only on 1 out of 130 sites.